lib/order.rb

```diff
@@ -0,0 +1,3 @@
+class Order
+  attr_accessor :id, :customer_name, :order_date, :item_id
```

Comment:
I recommend you to delete item_id from the orders table and add order_id to the items table.